src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.CompileToAssembly.Tests.cs

```diff
@@ -33,5 +38,34 @@ public void CompileToAssembly_PNSE()
                 [new CustomAttributeBuilder(typeof(AssemblyCompanyAttribute).GetConstructor([typeof(string)]), new[] { "TestCompany" })],
                 "resourceFile"));
         }
+
+        [ConditionalFact(nameof(IsDebug))]
+        public void CompileToAssembly_SimpleUseInDebug()
+        {
+            (RegexCompilationInfo rci, string validInput, string invalidInput)[] regexes =
```

Comment:
Just curios, seems the `validInput` and `invalidInput` is not used/tested/asserted. Is the compiled assembly did not work for validation or that part is coming later?

Reply:
Yup, they're not currently used. I could remove them, but left them in as I'd like to revisit getting the test to actually run the regex. The problem right now is there are multiple aspects of the implementation that assume in-memory only, such as generating IL that accesses private types in corelib. At some point when I have some free cycles I'll look into how minimal I can make tweaks in order to be able to successfully run them; the downside to that is, with the primary purpose of this being diagnostic, changes that diverge from the code that's used by RegexOptions.Compiled makes it potentially harder to diagnose issues with it.